packages/unit/src/types.doubles.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="jest" />
+/// <reference types="vitest" />
+/// <reference types="@types/sinon" />
+
+import type { StubbedInstance } from '@suites/types.doubles';
+
+// Re-export the StubbedInstance type
+export type { StubbedInstance };
+
+// Export framework-specific mocked types
+export type JestMocked<T> = jest.Mocked<T>;
+export type VitestMocked<T> = import('@vitest/spy').Mocked<T>;
+export type SinonMocked<T> = import('sinon').SinonStubbedInstance<T>;
```

Comment:
> Adding those types here seems problematic since the unit package isn't linked to any dependency injection or mocking library adapter (e.g. @suites/types.lib). Is there a specific reason for including them?

Same here